pyaedt/edb_core/components.py

```diff
@@ -1213,7 +1215,7 @@ def add_rlc_boundary(self, component=None, circuit_type=True):
             return True
 
     @pyaedt_function_handler()
-    def _create_pin_group_terminal(self, pingroup, isref=False):
+    def _create_pin_group_terminal(self, pingroup, isref=False, term_name=None):
```

Comment:
Same comment here about needing to add the new parameter to the "Parameters" section and format this section to our standard.